modules/rostests/apitests/user32/KbdLayout.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Tests for Keyboard Layouts DLL files
+ * COPYRIGHT:   Copyright 2022 Stanislav Motylkov <x86corez@gmail.com>
+ */
+
+#include "precomp.h"
+#include <ndk/kbd.h>
+#include <strsafe.h>
+
+typedef PVOID (*PFN_KBDLAYERDESCRIPTOR)(VOID);
+
+static void testLayout(LPWSTR szFileName, LPWSTR szFilePath)
```

Comment:
```suggestion
static void testLayout(
    _In_ LPCWSTR szFileName,
    _In_ LPCWSTR szFilePath)
```
on either separate lines or one single line, i don't care here.